hw/chisel/src/main/scala/snax/streamer/DesignParams.scala

```diff
@@ -22,6 +22,9 @@ class StreamerParam(
     // transpose params
     val hasTranspose: Boolean = false,
 
+    // c broadcast params
+    val hasCBroadcast: Boolean = false,
+
```

Comment:
why not just `hasBroadcast`? I feel like this extension could be used in other places than gemm as well?

Reply:
In theory yes, but actually, only the gemm's streamer uses extensions, and the extensions are also designed for gemm (specific parameters etc).
We can improve it to look more generic later though.